manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -35,3 +35,34 @@ spec:
     matchLabels:
       component: apiserver
       provider: kubernetes
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: kube-apiserver
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+  - name: using-deprecated-apis
+    rules:
+    - alert: UsingDeprecatedAPIAppsV1Beta1
+      annotations:
+        message: A client in the cluster is using deprecated apps/v1beta1 API that will be removed soon.
+      expr: |
+        apiserver_request_count{group="apps",version="v1beta1"}
+      labels:
+        severity: warning
+    - alert: UsingDeprecatedAPIAppsV1Beta2
+      annotations:
+        message: A client in the cluster is using deprecated apps/v1beta2 API that will be removed soon.
+      expr: |
+        apiserver_request_count{group="apps",version="v1beta2"}
+      labels:
+        severity: warning
+    - alert: UsingDeprecatedAPIExtensionsV1Beta1
+      annotations:
+        message: A client in the cluster is using deprecated extensions/v1beta1 API that will be removed soon.
+      expr: |
+        apiserver_request_count{group="extensions",version=~"v1beta1",resource!~"(?:ingresses|)",client!~".*/kube-controller-manager"}
```

Comment:
Any particular reason you do `=~` when matching version here? I doubt it's required.

Reply:
not really, just wanted to set pattern for easily adding another one if need be

Reply:
oh, you mean version - that's a leftover from where it was matching v1beta2 as well before I made it separate alert, will fix it tomorrow

Reply:
fixed